src/components/ChatEntry.js

```diff
@@ -1,22 +1,50 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
 
 const ChatEntry = (props) => {
+  const id = props.id;
+  const sender = props.sender;
+  const body = props.body;
+  const timeStamp = props.timeStamp;
+  const liked = props.liked;
+  const updateLiked = props.updateLiked;
+  const display = sender === 'Vladimir' ? 'local' : 'remote';
+  const heart = liked === true ? '❤️' : '🤍';
+  const updateColor = props.updateColor;
+  const color = updateColor;
+
+  const handleLiked = () => {
+    console.log('handleLiked called');
+    updateLiked(id);
+  };
```

Comment:
This works well! As an alternative, can you consider how you could do this as an anonymous function on line 31?